secrets

```diff
@@ -0,0 +1,4 @@
+AWS_KEY=AKIA1835sb
+password="ABC123DOREMI"
+credentials="@#ubdfbk@T#"
+password2="VAGADFAFASIN"
```

Comment:
### :warning: Aqua detected sensitive data in your code  
**Category:** Code Generic   
**Description:** Code Generic Credential   
**Severity:** HIGH   
**Match:** password2="************"  
[This comment was created by Aqua Pipeline]